packages/components/inputs/radio-input/src/radio-group.tsx

```diff
@@ -10,15 +10,12 @@ import {
 } from 'react';
 import { warning, filterDataAttributes } from '@commercetools-uikit/utils';
 import Constraints from '@commercetools-uikit/constraints';
-import Stack, { type TStackProps } from '@commercetools-uikit/spacings-stack';
-import Inline from '@commercetools-uikit/spacings-inline';
+import Stack from '@commercetools-uikit/spacings-stack';
+import Inline, {
+  type TInlineProps,
+} from '@commercetools-uikit/spacings-inline';
 import Option from './radio-option';
 
-const directionWrapper = {
-  stack: Stack,
-  inline: Inline,
-};
```

Comment:
This apparently wasn't used, and it was causing the issue (see screenshot).

Removed now.